exporter/prometheusexporter/accumulator_test.go

```diff
@@ -226,9 +230,10 @@ func TestAccumulateMetrics(t *testing.T) {
 			ilm2 := resourceMetrics2.ScopeMetrics().AppendEmpty()
 			ilm2.Scope().SetName("test")
 			tt.metric(ts2, 21, ilm2.Metrics())
+
 			tt.metric(ts1, 13, ilm2.Metrics())
 
-			a := newAccumulator(zap.NewNop(), 1*time.Hour).(*lastValueAccumulator)
+			a := newAccumulator(zap.NewNop(), 1*time.Hour, nil).(*lastValueAccumulator)
```

Comment:
If we remove the pointer, then we can't use nil of course. But we can call Noop telemetry 🤔 There are a few examples in the codebase of other components using this strategy